### ISSUE TITLE
Problem with training  

### ISSUE THREAD

**Issue by Doanhdz** [KG-TSI-Civil/CrackSAM#4]
First thanks for your great work. I have a question that did you use prompt when training your model ?

**Comment by KG-TSI-Civil** [comment]
No, I didn't use prompt. If you are interested in the application of prompt-guided SAM in civil engineering, I can recommend some interesting works (although they may not necessarily be open-sourced):
https://doi.org/10.1111/mice.13310
https://doi.org/10.1016/j.autcon.2024.105384
